package.json

```
@@ -11,14 +11,16 @@
     "start": "node dist/",
     "dev": "nodemon --exec babel-node src/",
     "dev:debug": "nodemon --exec babel-node --inspect=0.0.0.0:9229 src/index.js",
-    "lint": "eslint src --config .eslintrc.js",
-    "test": "nyc  --reporter=text-lcov yarn run test:jest",
+    "lint": "eslint src --config .eslintrc.js --fix",
```

please do not run lint with fix as it is breaking the build pipleline and is annoying as hell, if you need it, run it menually with that parameter.
```suggestion
    "lint": "eslint src --config .eslintrc.js",
```

---

Ok got it!